app/views/api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,9 @@
+### 9th June 2020
+
+New attributes:
+
+- `ApplicationAttributes` now has a unique `application_reference` attribute of type string.
```

Comment:
Sorry, one more thing! This attr isn't unique as the candidate may select two courses at the same provider, which would result in two applications coming over the API with the same application reference.